torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11211,6 +11211,7 @@ def ref_pairwise_distance(input1, input2):
            aten_name='conv_transpose1d',
            aliases=('conv_transpose1d',),
            dtypes=floating_types_and(torch.int64),
+           dtypesIfCPU=floating_types_and(torch.int64, torch.bfloat16),
```

Comment:
Why are you restricting the test dtypes here?

Reply:
conv_transpoed2d is enabled with bfloat16 with this pull request so that conv_transposed1d is enabled with bfloat16 ' as well since 3d input is unsqueezed to 4d in Convolution.cpp